### ISSUE TITLE
Add chord progressions highlights.

### ISSUE THREAD

**Issue by ca3games** [LMMS/lmms#5354]
## Enhancement Summary
Basically add some yellow ghost notes among several bars on the piano roll to indicate a chord, allowing you to see whatever chord progression you want in the measure.

#### Justification
It would be very helpful to be able to highlight common chord progressions in the piano roll.

#### Mockup
Just right click to highlight a chord and the measure or bar will become yellow, usign the same scale hightlight code.
Maybe add some basic button with a message asking how long it should the chord progression should be and what chords it should contain, maybe a simple integer next to the chord to indicate how long the chord should be.

**Comment by Veratil** [comment]
Currently if you right click on the piano keys you and you have a chord selected in the chord combobox, you can mark a chord highlight.

**Comment by musikBear** [comment]
> It would be very helpful to be able to highlight common chord progressions in the piano roll.

yak... 
+ oh no 'Common' progression -What are they?
An idea that is flawed on almost every level
* That would lead to boooring projects
* You remove the intuitive aspect of composing
* ...Can be done already..... -and even though a *hate* the idea, i will tell you how, because you would appearently like it..
Just 
* open an empty project
* Enter a chord progeression you would like in your library
* go to File| save as midi
* Name and save the progression logically in a library-folder

Whenever you want a specific progression, just load it with
* File| import and find the progression you want in your library

I still do not like it....

**Comment by DouglasDGI** [comment]
@ca3games Can you explain why this is needed if #4575 exists?

**Comment by ca3games** [comment]
> @ca3games Can you explain why this is needed if #4575 exists?

Sure, my idea is not super needed if you can simply drop a backdrop MIDI with some chord progression as guide.

Maybe I could change my request to being able to change the color of some ghost MIDI channel to another color, maybe yellow, so I can drop a MIDI as guide like musikBear suggested.

**Comment by Veratil** [comment]
@ca3games For some reason when I read this first and replied I wasn't thinking straight... so you want something similar to how Captain Chords works (the header area of it)?

**Comment by ca3games** [comment]
> @ca3games For some reason when I read this first and replied I wasn't thinking straight... so you want something similar to how Captain Chords works (the header area of it)?

Dunno how captain chords does it, but I meant more like those FL studio youtube tutorial when they lay out the chord progression before starting a song.
It's more like a ruler guideline helper kind of thing.

**Comment by DouglasDGI** [comment]
> Maybe I could change my request to being able to change the color of some ghost MIDI channel to another color, maybe yellow, so I can drop a MIDI as guide like musikBear suggested.

In that case, it should be put into a separate issue and this should be closed as a duplicate.

**Comment by Veratil** [comment]
> 
> 
> > @ca3games For some reason when I read this first and replied I wasn't thinking straight... so you want something similar to how Captain Chords works (the header area of it)?
> 
> Dunno how captain chords does it, but I meant more like those FL studio youtube tutorial when they lay out the chord progression before starting a song.
> It's more like a ruler guideline helper kind of thing.

Can you share a video that has it?

**Comment by ca3games** [comment]
> > > @ca3games For some reason when I read this first and replied I wasn't thinking straight... so you want something similar to how Captain Chords works (the header area of it)?
> > 
> > 
> > Dunno how captain chords does it, but I meant more like those FL studio youtube tutorial when they lay out the chord progression before starting a song.
> > It's more like a ruler guideline helper kind of thing.
> 
> Can you share a video that has it?

Dunno, look up any FL chord progression song tutorial.
They always need to layout the chord progression as a guideline.

**Comment by musikBear** [comment]
@ca3games wrote
> look up any FL chord progression song tutorial. They always need to layout the chord progression as a guideline.

Yes! Because FL-studio does not have Scales, as we have in LMMS. I believe its a FL-enterprice feature, but a lot of FL'ers draw the scale, before they start to insert notes, maybe thats what you mean. A chord-progression library can be made already, as i outlined

**Comment by zonkmachine** [comment]
@ca3games Basically this is implemented already. You can select a chord or scale and highlight the corresponding notes in the Piano Roll. Watch @musikBear's post above.

1. Select a scale or chord in the menu.
2. Right click the keyboard to the left and choose to highlight a scale or chord. Done...

I'm closing this feature request.
